ui/component/uriIndicator/view.jsx

```diff
@@ -67,10 +70,11 @@ class UriIndicator extends React.PureComponent<Props> {
             <Tooltip label={<ClaimPreview uri={channelLink} type="tooltip" placeholder={false} />}>{children}</Tooltip>
           )
         : 'span';
-
+      /* to wrap the UriIndicator element around other DOM nodes as a button */
+      const content = children ? children : (<Wrapper>{inner}</Wrapper>);
```

Comment:
This should probably just be `children || <Wrapper>{inner}</Wrapper>`

Reply:
done